### PR TITLE
:truck: Replace string by key translate fix #64

### LINKED ISSUE #64: Translation strings
Use translation strings rather than a mix of French and English

### PR DESCRIPTION
**Pull request by issa-diallo**
- Add react-i18next for key translate
- Change all title by key

### CODE REVIEW

File: src/i18n.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import i18n from 'i18next'
+import { initReactI18next } from 'react-i18next'
+import LanguageDetector from 'i18next-browser-languagedetector'
+// import HttpBackend from 'i18next-http-backend' // *** added this ***
+import { BACKEND_URL } from './services/constants'
+
+i18n
+//   .use(HttpBackend) // *** added this ***
```
Comment (AndreMiras) [inline src/i18n.ts:8]: Let's remove the commented code

File: src/i18n.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  // learn more: https://github.com/i18next/i18next-browser-languageDetector
+  .use(LanguageDetector)
+  // pass the i18n instance to react-i18next.
+  .use(initReactI18next)
+  // init i18next
+  // for all options read: https://www.i18next.com/overview/configuration-options
+  .init({
+    debug: true,
+    fallbackLng: 'fr',
+    supportedLngs: ['en', 'es', 'fr'], // *** added this ***
```
Comment (AndreMiras) [inline src/i18n.ts:19]: Comment node needed
Also are you sure "es" is available already?

**Review by AndreMiras — APPROVED** [review]
Pretty cool!
I've added some minor comments

File: src/i18n.ts (hunk 2 above)
Comment (issa-diallo) [reply src/i18n.ts:19]: I 've remove "es"

**Pull request merged**